src/nomad_simulations/physical_property.py

```diff
@@ -0,0 +1,231 @@
+        """,
+    )
+
+    source = Quantity(
+        type=MEnum('simulation', 'measurement', 'analysis'),
+        default='simulation',
+        description="""
+        Source of the physical property. Example: an `ElectronicBandGap` can be obtained from a `'simulation'`
+        or in an `'measurement'`.
+        """,
+    )
+
+    type = Quantity(
+        type=str,
+        description="""
+        Type categorization of the physical property. Example: an `ElectronicBandGap` can be `'direct'`
+        or `'indirect'`.
+        """,
+        # ! add more examples in the description
+    )
+
+    label = Quantity(
+        type=str,
+        description="""
+        Label for additional classification of the physical property. Example: an `ElectronicBandGap`
+        can be labeled as `'DFT'` or `'GW'` depending on the methodology used to calculate it.
+        """,
+        # ! add more examples in the description
+    )
+
+    rank = DirectQuantity(
+        type=Dimension,
+        shape=['0..*'],
+        default=[],
+        name='rank',
+        description="""
+        Rank of the tensor describing the physical property. This quantity is stored as a Dimension:
+            - scalars (tensor rank 0) have `rank=[]` (`len(rank) = 0`),
+            - vectors (tensor rank 1) have `rank=[a]` (`len(rank) = 1`),
+            - matrices (tensor rank 2), have `rank=[a, b]` (`len(rank) = 2`),
+            - etc.
+        """,
+    )
+
+    variables = SubSection(sub_section=Variables.m_def, repeats=True)
+
+    value: Quantity = _placeholder_quantity
+
+    entity_ref = Quantity(
+        type=Entity,
+        description="""
+        Reference to the entity that the physical property refers to.
+        """,
+    )
+
+    physical_property_ref = Quantity(
+        type=Reference(SectionProxy('PhysicalProperty')),
+        description="""
+        Reference to the `PhysicalProperty` section from which the physical property was derived. If `physical_property_ref`
+        is populated, the quantity `is_derived` is set to True via normalization.
+        """,
+    )
+
+    is_derived = Quantity(
+        type=bool,
+        default=False,
+        description="""
+        Flag indicating whether the physical property is derived from other physical properties. We make
+        the distinction between directly parsed and derived physical properties:
+            - Directly parsed: the physical property is directly parsed from the simulation output files.
+            - Derived: the physical property is derived from other physical properties. No extra numerical settings
+                are required to calculate the physical property.
+        """,
+    )
+
+    is_scf_converged = Quantity(
+        type=bool,
+        description="""
+        Flag indicating whether the physical property is converged or not after a SCF process. This quantity is connected
+        with `SelfConsistency` defined in the `numerical_settings.py` module.
```

Comment:
I would mention `numerical_settings.py` in the description of `self_consistency_ref`